### PR TITLE
Chore ut resource configuration variable

### PR DESCRIPTION
**Pull request by eranelbaz**
### Issue & Steps to Reproduce / Feature Request
Tests tests tests

### CODE REVIEW

File: env0/resource_configuration_variable_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import (
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+func TestUnitConfigurationVariableResource(t *testing.T) {
+	resourceType := "env0_configuration_variable"
+	resourceName := "test"
+	resourceFullName := resourceAccessor(resourceType, resourceName)
```
Comment (razbensimon) [inline env0/resource_configuration_variable_test.go:12]:
```suggestion
	accessor := resourceAccessor(resourceType, resourceName)
```
to fit new structure

**Review by razbensimon — COMMENTED** [review]
You should split the cases to different Test methods.
see other exanples

File: env0/resource_configuration_variable_test.go (hunk 1 above)
Comment (eranelbaz) [reply env0/resource_configuration_variable_test.go]: 3d5100ff

**Comment by eranelbaz** [comment]
@razbensimon 2afad039

**Comment by razbensimon** [comment]
merge/ rebase first, but looks good 👏

**Review by razbensimon — APPROVED** [review]

File: env0/resource_configuration_variable_test.go [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+	}
+
+	runUnitTest(t, createTestCase, func(mock *client.MockApiClientInterface) {
+		mock.EXPECT().ConfigurationVariableCreate(configVar.Name, configVar.Value, false, client.ScopeGlobal, "", client.ConfigurationVariableTypeEnvironment,
+			nil).Times(1).Return(configVar, nil)
+		mock.EXPECT().ConfigurationVariables(client.ScopeGlobal, "").Times(1).Return([]client.ConfigurationVariable{configVar}, nil)
+		mock.EXPECT().ConfigurationVariableDelete(configVar.Id).Times(1).Return(nil)
+	})
+}
+func TestUnitConfigurationVariableResourceCreateWrongType(t *testing.T) {
```
Comment (razbensimon) [inline env0/resource_configuration_variable_test.go:45]:
```suggestion
func TestUnitConfigurationVariableResource_CreateWrongType(t *testing.T) {
```

do it for all

Comment (eranelbaz) [reply env0/resource_configuration_variable_test.go]: 49d5f7cb

**Comment by eranelbaz** [comment]
@razbensimon 79dfbab6 see commit message

**Comment by eranelbaz** [comment]
@razbensimon 79dfbab6 see commit message

**Pull request merged**